src/styles/main.scss

```diff
@@ -1,3 +1,180 @@
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+p {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  // margin: 0 auto 74px;
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.is-active,
+.nav__link {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  margin: 0 10px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.is-active::after,
+.nav__link::after {
+  content: "";
+  display: block;
+  position: absolute;
+  left: 0;
+  width: 0;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+  transition: width 0.3s ease-out;
+  margin-top: 18px;
+}
+
+.is-active,
+.is-active::after,
+.nav__link:hover,
+.nav__link:hover::after {
+  color: #00acdc;
+  width: 100%;
+}
+
+.main {
+  &__wrapper {
+    margin-bottom: 50px;
+  }
+
+  &__cards {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    margin: 0 auto;
+    padding: 0 50px;
+  }
+}
+
+.product-card {
+  width: 200px;
+  height: 404px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-top: 50px;
+}
+
+.product-card__img {
+  object-fit: contain;
+  width: 166px;
+  height: 134px;
+}
+
+.product-card__name {
+  margin-top: 36px;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+}
+
+.product-card__code {
+  font-size: 10px;
+  margin: 0;
+  font-weight: normal;
+  line-height: 14px;
+  color: #616070;
+}
+
+.product-card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  cursor: pointer;
+  justify-content: center;
+  padding-top: 1px;
+  padding-left: 2px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars .stars__star:last-child {
+  background-image: url(../images/star.svg);
+}
+
+.product-card__rating-value {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 22px 0 16px;
+}
+
+.product-card__price-text {
+  font-size: 12px;
+  color: #616070;
+}
+
+.product-card__price-cost {
+  font-weight: 700;
+  font-size: 16px;
+  color: #060b35;
+}
+
+.product-card__button {
+  display: flex;
+  height: 40px;
+  align-items: center;
+  justify-content: center;
+  background: #00acdc;
+  border-radius: 5px;
+  line-height: 40px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.product-card__button:hover {
+  background: #fff;
+  color: #00acdc;
+  border: 1px solid #f3f3f3;
+  text-decoration: none;
```

Comment:
This style is not changing, it is useless under `:hover` selector

Reply:
this works when hovering over the buy button